### PR TITLE
updated responsive props for border radius

### LINKED ISSUE #16491: Fix: Box component broken responsive props for border radius
### Description
The responsive props for the `borderRadius` prop aren't working (and may have never worked mb! 🤦 ). The classNames are added correctly in the html but it seems the sass isn't properly generated. It might be worth adding border radius to `ui/css/design-system/attributes.scss` something like:

```scss
// in ui/css/design-system/attributes.scss

$border-radius: (
  none: 0,
  xs: 2px,
  md: 4px,
  lg: 8px,
  xl: 12px,
  full: 50%,
  pill: 9999px
);
```

## Screencast

Story below has responsive props for borderRadius on all boxes. The classNames are being added to the html but the styles are not being generated

https://user-images.githubusercontent.com/8112138/201769893-3744a806-8608-49ec-8588-e2e4fa8ff5c6.mov

While we are at it we should be consistent with pixels. This update should consolidate these values anyway 

Screen Shot 2022-11-14 at 3 39 10 PM

### PR DESCRIPTION
**Pull request by NidhiKJha**
This PR is to update responsive props for border-radius in Box

* Fixes #16491 

## Manual Testing Steps

`yarn test:unit:jest .ui/components/ui/box/box.test.js   `

- Go to the latest storybook build
- Search for Box in UI
- Check the stories, controls, and documentation

## Pre-Merge Checklist

- [x] PR template is filled out
- [x] **IF** this PR fixes a bug, a test that _would have_ caught the bug has been added
- [x] PR is linked to the appropriate GitHub issue
- [x] PR has been added to the appropriate release Milestone

### + If there are functional changes:

- [x] Manual testing complete & passed
- [x] "Extension QA Board" label has been applied

### CODE REVIEW

**Review by georgewrmarshall — COMMENTED** [review]
Amazing! Could we add a story to the border radius stories with responsive props?

**Review by brad-decker — DISMISSED** [review]

**Review by georgewrmarshall — COMMENTED** [review]
Checked out your branch to see the story and it looks good! Just getting a propType error in the console

Screen Shot 2022-11-16 at 4 59 36 PM

File: ui/components/ui/box/box.stories.js [hunk 1]
```diff
@@ -583,7 +583,7 @@ export const ResponsiveProps = () => {
         padding={[2, 4]}
         gap={[2, 4]}
         display={['flex', null, null, 'none']}
-        flexDirection={['column', 'row']}
+        flexDirection={['column', 'column', 'row']}
```
Comment (georgewrmarshall) [inline ui/components/ui/box/box.stories.js:586]: Just realized we should probably be using the design-system consts here. Old mistake from me so my bad. But not blocking

e.g. `display={[DISPLAY.FLEX, null, null, DISPLAY.NONE]}`

**Review by georgewrmarshall — DISMISSED** [review]

File: ui/components/ui/box/box.stories.js [hunk 2]
```diff
@@ -582,8 +582,8 @@ export const ResponsiveProps = () => {
         marginBottom={[0]}
         padding={[2, 4]}
         gap={[2, 4]}
-        display={['flex', null, null, 'none']}
-        flexDirection={['column', 'row']}
+        display={[DISPLAY.FLEX, null, null, DISPLAY.NONE]}
+        flexDirection={['column', 'column', 'row']}
```
Comment (georgewrmarshall) [inline ui/components/ui/box/box.stories.js:586]: Oh could do the same here :) 

```suggestion
        flexDirection={[FLEX_DIRECTION.COLUMN, FLEX_DIRECTION.COLUMN, FLEX_DIRECTION.ROW]}
```

**Review by georgewrmarshall — COMMENTED** [review]
one small suggestion :)

**Review by georgewrmarshall — APPROVED** [review]

**Review by garrettbear — APPROVED** [review]

**Pull request merged**